### PR TITLE
Recordings UX improvements

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

From my own dogfooding yesterday, I discovered some UI/UX stuff on the recordings experience worth improving.
* Moves recordings table to `LemonTable`
* Top bar was too tight
* Padding on close button was off
* Recording time ago should show the timezone conversion
* Adjust copy on "sessions" (including page title)

## How did you test this code?
- Tested UI with `new-sessions-player-events-list` on & off

### CODE REVIEW

File: frontend/src/lib/utils.tsx [hunk 1]
```diff
@@ -507,7 +507,7 @@ export function humanFriendlyDuration(d: string | number | null | undefined, max
 
     const dayDisplay = days > 0 ? days + 'd' : ''
     const hDisplay = h > 0 ? h + 'h' : ''
-    const mDisplay = m > 0 ? m + 'min' : ''
+    const mDisplay = m > 0 ? m + 'm' : ''
```
Comment (paolodamico) [inline frontend/src/lib/utils.tsx:510]: For consistency

File: frontend/src/lib/utils.tsx [hunk 2]
```diff
@@ -524,12 +524,15 @@ export function humanFriendlyDiff(from: dayjs.Dayjs | string, to: dayjs.Dayjs |
     return humanFriendlyDuration(diff)
 }
 
-export function humanFriendlyDetailedTime(date: dayjs.Dayjs | string | null, withSeconds: boolean = false): string {
+export function humanFriendlyDetailedTime(
+    date: dayjs.Dayjs | string | null,
+    withSeconds: boolean = false,
+    formatString: string = 'MMMM DD, YYYY h:mm'
```
Comment (paolodamico) [inline frontend/src/lib/utils.tsx:530]: Previously we had a bug with `Do`

File: frontend/src/scenes/sceneLogic.ts [hunk 3]
```diff
@@ -190,7 +190,7 @@ export const sceneLogic = kea<sceneLogicType>({
         },
         setScene: ({ scene, scrollToTop }, _, __, previousState) => {
             posthog.capture('$pageview')
-            setPageTitle(identifierToHuman(scene || ''))
+            setPageTitle(sceneConfigurations[scene]?.name || identifierToHuman(scene || ''))
```
Comment (paolodamico) [inline frontend/src/scenes/sceneLogic.ts:193]: We now display the custom-defined scene name if available

**Review by rcmarron — APPROVED** [review]
This looks great! Thanks for taking care of it!

**Pull request merged**